samples/nrf9160/nrf_cloud_rest_agps/Kconfig

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "nRF Cloud REST A-GPS API Sample"
```

Comment:
Now that the sample doesn't use the GPS driver, there needs to be handling for the GNSS LNA configuration (AT%XMAGPIO and AT%XCOEX0). This would really belong in the board configuration, but it's not always possible to send AT commands from there currently. A better solution is in the works, but meanwhile you can have a look how the configuration has been done for example in the GPS sample (CONFIG_GPS_SAMPLE_ANTENNA*, CONFIG_GPS_SAMPLE_AT_MAGPIO, CONFIG_GPS_SAMPLE_AT_COEX0 and related code).

Reply:
OK.